### PR TITLE
Meshery icons

### PR DESCRIPTION
**Pull request by Shivam-AfA**
**Description**

This PR fixes **integration icons at https://layer5.io/cloud-native-management/meshery/integrations** and also #3579 

**Notes for Reviewers**
Some of the meshery integration icons at the above mentioned link were either incorrect or were not following dark and light theme color schemes.

**The icons changed include:**
- **Azure Application Gateway** - white
- **Cert Manager icon** - (Both color and white icons were incorrect)
- **Consul** - color
- **ContainerSSH** - white
- **Citrix Service Mesh** **(cpx)** - (Both color and white icons were incorrect)
- **JFrog Distribution** - (Both color and white icons were incorrect)
- **Elastic Search Operator** - white
- **Emissary Ingress** - Color and white icons needed to be interchanged.
- **Fluent Operator** - (Both icons were incorrect)
- **Flux** - color
- **Kube-rs** - color
- **Kubernetes** - color
- **Kubesphere** - white
- **Linux** - color
- **Meshery** - color
- **Mimir Distributed** - white
- **Nginx Ingress** - color
- **Nginx** - color
- **OpenEBS** - white
- **Oras** - white
- **Spiffe** - color
- **Terraform** - white
- **Windows** - white

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/integrations/cert-manager-csi-driver-cacerts/index.mdx [hunk 1]
```diff
@@ -25,4 +25,4 @@ published: true
 </p>
 <p>
     Looking for best practice cloud native design and deployment best practices? Choose from thousands of pre-built components in MeshMap. Choose from hundreds of ready-made design patterns by importing templates from Meshery Catalog or use our low code designer, MeshMap, to create and deploy your own cloud native infrastructure designs.
-</p>
```
Comment (Shivam-AfA) [inline src/collections/integrations/cert-manager-csi-driver-cacerts/index.mdx:28]: Kindly ignore this change! 😅

**Comment by l5io** [comment]
🚀 Preview for commit 8e59d6f3860e937d150b3384d8b5aed79d7861f5 at: https://63af6188d043e616fb132deb--layer5.netlify.app

**Comment by Shivam-AfA** [comment]
@leecalcote @UtkarshMishra12 @GaganpreetKaurKalsi Did you check this PR out?

**Review by GaganpreetKaurKalsi — COMMENTED** [review]
**ContainerSSH - white** - Maybe just make the black background transparent in color icon to create white one. Or if we want to remove the red band as well, we can change it to white.

Others look fine to me. Also we can replace the white icon for Github with the one for Arc.

**Comment by Shivam-AfA** [comment]
> **ContainerSSH - white** - Maybe just make the black background transparent in color icon to create white one. Or if we want to remove the red band as well, we can change it to white.
> 
> Others look fine to me. Also we can replace the white icon for Github with the one for Arc.

Okay I'll tweak the ContainerSSH icon and thanks for pointing out the Arc icon as well. However, I could not find a logo for the Arc icon, so it would be great if you could help me out with it! @GaganpreetKaurKalsi

**Comment by Abiey2579** [comment]
@Shivam-AfA for some of the Images causing us headache I figure out we can use

`filter: brightness(0) invert(1);` OR `filter: grayscale(1)`

Though am sure the first one will work just fine.

I have even tried it out, by inspecting the elements, and it works.

**Comment by Shivam-AfA** [comment]
> @Shivam-AfA for some of the Images causing us headache I figure out we can use
> 
> `filter: brightness(0) invert(1);` OR `filter: grayscale(1)`
> 
> Though am sure the first one will work just fine.
> 
> I have even tried it out, by inspecting the elements, and it works.

Thanks @Abiey2579. I'll try to implement this!

**Comment by GaganpreetKaurKalsi** [comment]
@Shivam-AfA Need to fix the devfile logo as well, the size is too small.

**Comment by Shivam-AfA** [comment]
> @Shivam-AfA Need to fix the devfile logo as well, the size is too small.

Noted. @GaganpreetKaurKalsi

**Comment by Shivam-AfA** [comment]
Hey @GaganpreetKaurKalsi, I have corrected the ContainerSSH - white and the Devfile icons. However, I did mess up some commits in the process (as you can surely notice 😅) Do you know how to fix this?

**Comment by GaganpreetKaurKalsi** [comment]
> Hey @GaganpreetKaurKalsi, I have corrected the ContainerSSH - white and the Devfile icons. However, I did mess up some commits in the process (as you can surely notice 😅) Do you know how to fix this?

I see. It happens when we do force push I guess. I had this same mess of commits in my PR earlier. No need to worry I guess.  Just take care of the failing build. There is some issue with some file I presume.

**Comment by Shivam-AfA** [comment]
> > Hey @GaganpreetKaurKalsi, I have corrected the ContainerSSH - white and the Devfile icons. However, I did mess up some commits in the process (as you can surely notice 😅) Do you know how to fix this?
> 
> I see. It happens when we do force push I guess. I had this same mess of commits in my PR earlier. No need to worry I guess.

Okay. And BTW this PR is just to see the icon changes, eventually we have to update the spreadsheet, not merge this PR.

**Comment by Shivam-AfA** [comment]
@leecalcote @GaganpreetKaurKalsi Can these icons be updated in the spreadsheet? The inconsistent honeycomb of meshery integrations doesn't look quite good.

**Comment by leecalcote** [comment]
> @leecalcote @GaganpreetKaurKalsi Can these icons be updated in the spreadsheet? The inconsistent honeycomb of meshery integrations doesn't look quite good.

@Shivam-AfA, yes, please. With your edit access, please do proceed (with care).

**Comment by Shivam-AfA** [comment]
Sure @leecalcote 😎

**Pull request closed without merging**